tests/retryable-reads/retryable-reads_error-001.phpt

```diff
@@ -0,0 +1,33 @@
+--TEST--
+Retryable reads: failure when disabled
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_libmongoc_crypto(); ?>
+<?php skip_if_server_version('<', '3.6'); ?>
+<?php skip_if_no_failcommand_failpoint(); ?>
+<?php skip_if_not_clean(); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(URI, ['retryReads' => false]);
+
+configureFailPoint($manager, 'failCommand', ['times' => 1], ['failCommands' => ['find'], 'closeConnection' => true]);
+
+throws(
```

Comment:
Asserting the exception is good, but I think it makes sense to still use APM and assert that only a single command is executed, too. This test could conceivably pass if `times: 2` was specified in the fail point and `retryReads` was enabled.

Reply:
Both failure tests now include a command observer to assert against the number of commands executed.